test/features/ReasonsToDonate.ts

```diff
@@ -12,7 +12,7 @@ const expectContainsReasonsToDonateDialogue = async ( wrapper: VueWrapper<any> )
 };
 
 const expectScrollsToFormWhenCallToActionIsClicked = async ( wrapper: VueWrapper<any>, pageScroller: PageScroller ): Promise<any> => {
-	await wrapper.find( '.wmde-banner-reasons-to-donate-link' ).trigger( 'click' );
+	await wrapper.find( '.ten-good-reasons-sticker-text' ).trigger( 'click' );
```

Comment:
changing the test like this will make the Desktop DE 14 tests fail because they rely on them.

Maybe you can duplicate the tests and have "sticker"  vs. "link" as a click source

Reply:
Thank you for pointing it out. I added new tests for 'sticker'.
I did not rename old tests to be 'link' tests, because then `desktop-de-14` and `mob-de-09` would have to be changed to reflect the rename changes.